content/battery.md

```diff
@@ -19,167 +19,155 @@ section: hardware
+The system will default on each startup to the "Balanced" setting, which is usually preferred even when connected to AC power as it helps keep the system running cool and quiet. When maximum performance is needed, the "High Performance" setting will uncap everything and let the system run as hot and power-hungry as it wants, which can dramaticaly reduce battery life and may also increase the noise made by the system fan(s).
 
-To install TLP, run this command:
+The "Battery Life" setting will do a number of things to improve battery life, including reduce the screen brightness (which can be turned back up as described above if desired), reduce the maximum speed of the CPU (and any GPUs if present), aggressively control how applications are allowed to use the CPU and GPU, and much more under the hood.
 
-```bash
-sudo apt install tlp tlp-rdw --no-install-recommends
-```
+## Switchable graphics settings
 
-TLP will take effect upon restart. To see current configuration settings, run this command:
+The settings for the graphics modes on laptops that have switchable Nvidia graphics are outlined in detail [on their own support document](https://support.system76.com/articles/graphics-switch-pop), but to summarize:
 
-```bash
-sudo tlp-stat
-```
+* Integrated Intel graphics mode will provide increased battery life at the cost of 3D performance, and some laptops will not support external displays in that mode.
+* Hybrid graphics allow you to choose to use the more powerful Nvidia GPU when needed, but applications will default to using the lower-power Intel graphics.
+* Nvidia graphics mode runs everything on the Nvidia GPU for maximum perormance, and will use the most battery.
+* Compute mode reserves the Nvidia GPU for tasks that don't output graphics to the screen, like machine learning or data crunching processes.
 
-The program is highly configurable by editing the settings file. Run this command to edit the file:
+Switching graphics modes does require restarting the laptop, which will use up some battery life in the process, so it's recommended that the graphics mode be switched before disconnecting from the AC adapter.
 
-```bash
-sudo gedit /etc/default/tlp
-```
+## Disabling unused wireless devices
 
-Starting with <u>TLP</u> 1.3 the default configuration file has changed so the command for that version is below:
+WiFi and Bluetooth are wireless technologies that use up a small amount of power even when they're not being used, as they monitor for wireless network activity. If you're not using them, disabling either WiFi or Bluetooth can improve battery life, and the "Airplane Mode" setting will disable both for maximum savings. They can be disabled individually via the Wi-Fi and Bluetooth panels in the Settings application by toggling the switch at the top of the window, and Airplane Mode is available underneath it on the Wi-Fi settings panel:
```

Comment:
Very minor, but we have both Wi-Fi and WiFi here. Let's go with Wi-Fi since that's what's called out in the menu item.

Menu items should be in backtics.

Reply:
Done, will be updated on the next git push.